content/applications/sales/crm/acquire_leads/generate_from_website.rst

```diff
@@ -1,78 +1,104 @@
-===========================================================
-Generate leads/opportunities from your website contact page
-===========================================================
-
-Automating the lead/opportunity generation will considerably improve
-your efficiency. Any visitor using the contact form on your website will
-create a lead/opportunity in the pipeline.
-
-Use the contact us on your website
-==================================
+============================
+Generate leads/opportunities
```

Comment:
Maybe this file should be renamed to `generate_leads.rst` to avoid confusion?

Reply:
Yes, absolutely. Will do!